tools/system_libs.py

```diff
@@ -1021,7 +1022,6 @@ class libhtml5(Library):
 class libpthread(AsanInstrumentedLibrary, MuslInternalLibrary, MTLibrary):
   name = 'libpthread'
   depends = ['libc']
-  cflags = ['-O2']
```

Comment:
This library may also lose inlining. But maybe it's less worrying here... not sure though. I'd recommend keeping the old flags.

Reply:
This seems kind of arbirary (and I'm not sure it was deliberate).   I don't think inlining is any more desirable for the the code in the pthread subdirectory that any other sub-directory is it?